### DISCUSSION TITLE
Let's make both Github Copilot VSCode extensions compatible with https://vscode.dev and https://github.dev

### DISCUSSION THREAD

**Discussion by beingminimal** [community/community#15533] [Copilot Conversations] [answered]
Hey everyone you all are requested to vote for this.

Make both VSCode extensions of Github Copilot compatible with https://vscode.dev/ and https://github.dev/
Please consider this as DX improvement and give some priority for it.

Here is more info
https://aka.ms/vscode-web-extensions-guide
https://code.visualstudio.com/api/extension-guides/web-extensions

Thanks

![Screenshot 2022-04-25 09 08 53](https://user-images.githubusercontent.com/84431724/165017077-1fe68111-8e40-44d0-af35-1b9c0bf94679.png)

![Screenshot 2022-04-25 09 08 57](https://user-images.githubusercontent.com/84431724/165017058-ebcc3b94-602b-4e24-a832-2ef3372c8b72.png)

**Comment by ytrezq** [comment]
Hello, since you have an access, could you please use it for me please?

I need to convert Volfs files from an old wsl install to DrvFs https://github.com/0xbadfca11/lxsstat/wiki/WSL-filesystem given a Windows file path. but I ve absolutely no ideas on how to parse the relevant ntfs xattrs.

**Reply by superdba111** [reply]
please grant me access permissions on github copilot, thanks

**Comment by hsenag** [comment, marked as answer]
Hi, thanks for the interest! This is something that's on our radar but we don't have an eta at present.

**Reply by sxwebster** [reply]
Now that copilot is available commercially this would be very nice to see. Certainly for anyone wanting to program or update markdown on the move (ipad, chromebook, etc)

**Comment by dililidilili** [comment]
I'm  on the waitlist  for two weeks already! please ~
